build_player_info.py

```diff
@@ -0,0 +1,40 @@
+import csv
```

Comment:
This is just a little script I wrote to generate a json file from your csv. It doesn't need to stick around, unless we would be adding new players on a regular basis. What do you think of my python? 😄